Automattic/wp-calypso
# Gutenboarding importer flow: improve user experience

#### Changes proposed in this Pull Request

Changes introduce a few improvements:

- Integrated spinner on the scanning screen
- Improved quality of preview image
- Improved UX on showing the error message on the capture screen

#### Testing instructions

Manual test:
- Go to `/start/importer` page
- Enter invalid URL, then hit the return key
- Error message will be shown
- Enter valid URL, then hit the return key
- Scanning screen contains a spinner
- Then, you can check the quality of Mshot image


Screenshot:
![Screen Capture on 2021-10-26 at 13-25-53](https://user-images.githubusercontent.com/1241413/138868757-feb43f17-166e-416d-b6fd-bed1219777c8.gif)
